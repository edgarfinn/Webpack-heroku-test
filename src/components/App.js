import React, {Component} from 'react';
import AlbumList from '../containers/AlbumList/album_list';
import AlbumDetail from '../containers/AlbumDetail/album_detail';
import NewAlbumForm from '../containers/NewAlbumForm/new_album_form';

class App extends Component {
  render() {
    return (
      <div>
        <h1>Welcome To React Custom!</h1>
        <AlbumList />
        <AlbumDetail />
        <NewAlbumForm />
      </div>
    )
  }
}


export default App;
