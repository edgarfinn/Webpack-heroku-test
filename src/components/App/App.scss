@import '../../Sass/variables';

.page-header {
  font: $font-stack;
  color: $primary;
}
