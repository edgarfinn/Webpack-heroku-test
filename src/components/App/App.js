import React, { Component } from 'react';
import './App.scss';

import PageHeader from './PageHeader/Page_header';
import SearchForm from '../../Containers/SearchForm/Search_form';

export class App extends Component {
  render() {
    return (
      <div className="app">
        <PageHeader />
        <SearchForm />
      </div>
    )
  }
}

export default App;
