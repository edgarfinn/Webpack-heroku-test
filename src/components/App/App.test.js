import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
Enzyme.configure({ adapter: new Adapter() });

import { shallowToJson } from 'enzyme-to-json';

import {App} from './App';

describe('<App />', () => {
  it('renders correctly', () => {

    const wrapper = shallow(<App />)
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  })
})
