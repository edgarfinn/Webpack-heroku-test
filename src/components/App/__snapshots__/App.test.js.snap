// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders correctly 1`] = `
<div>
  <Component />
  <Connect(AlbumList) />
  <Connect(AlbumDetail) />
  <ReduxForm />
</div>
`;
